custom-local-authenticator/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we change all the licensing to 2020

Reply:
Shall we change this to 2020? It is still 2019